solr/core/src/java/org/apache/solr/handler/admin/CoreAdminHandler.java

```diff
@@ -411,6 +413,8 @@ void call() throws Exception {
   public Collection<Api> getApis() {
     final List<Api> apis = Lists.newArrayList(coreAdminHandlerApi.getApis());
     // Only some core-admin APIs use the v2 AnnotatedApi framework
```

Comment:
is the above comment still true.

Reply:
Yep, even after this PR there are the `cores.core.Commands` and the cores.core.Commands.split` apispec files.